src/helics/core/BrokerFactory.cpp

```diff
@@ -83,6 +82,7 @@ namespace BrokerFactory {
 
     std::shared_ptr<Broker> create(core_type type, const std::string& configureString)
     {
+        static const std::string emptyString;
```

Comment:
Is there an advantage having a static const variable declared instead of using `std::string{}` as an argument?

Reply:
I am trying to figure that out.  As best as I can tell, there is a slight simplification in debug since a string isn't created.  In release mode it gets more complicated.  in some cases there are lifetime issues with static global strings.    In release mode it seems to vary in some cases the string is essentially optimized out depending on compiler.  In other cases it is not.    I think in HELICS 3 I am leaning to move to all locally constructed since the compilers are better and can optimize it out mostly.